lms/__init__.py

```diff
@@ -4,6 +4,17 @@
 """
 from __future__ import absolute_import
 
+# We monkey patch Kombu's entrypoints listing because scanning through this
+# accounts for the majority of LMS/Studio startup time for tests, and we don't
+# use custom Kombu serializers (which is what this is for). Still, this is
+# pretty evil, and should be taken out when we update Celery to the next version
+# where it looks like this method of custom serialization has been removed.
+#
+# FWIW, this is identical behavior to what happens in Kombu if pkg_resources
+# isn't available.
+import kombu.utils
+kombu.utils.entrypoints = lambda namespace: iter([])
```

Comment:
Question: Why in `__init__.py` instead of `test.py`?  Of course, by putting it here, it also optimizes LMS/CMS startup times, not just for tests.  But I'm asking since monkey-patching is generally considered a taboo, so was wondering if it should only be within test code.